src/System.Data.SqlClient/tests/FunctionalTests/SqlCredentialTest.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Security;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Data.SqlClient.Tests
+{
+    public static class SqlCredentialTest
```

Comment:
How about adding a test which tries to retrieve the attributes `UserId` and `Password` from the SqlCredential object?